example/README.md

```diff
@@ -1,2 +1,10 @@
-This project is a starting point for a Flutter application with a RevenueCat integration. For help 
-getting started with, view our [online documentation](https://docs.revenuecat.com).
+# Examples
+Our examples are located at `revenuecat_examples/`.
+
+### MagicWeather
+
+This project is the best showcase for how to use `purchases-flutter` for a subscriptions app.
```

Comment:
I think the right name is `purchases_flutter`

Reply:
good catch, the repo is with `-` but the package is `_`